test/verify/machineslib.py

```diff
@@ -1445,6 +1465,12 @@ def fill(self):
                 b.click("#start-vm") # TODO: fix this, do not assume initial state of the checkbox
             # b.set_checked("#start-vm", self.start_vm)
 
+            if self.choose_storage:
+                b.wait_visible("#choose-storage")
+                b.click("#choose-storage") # TODO: fix this, do not assume initial state of the checkbox
+                b.select_from_dropdown("#storage-pool-select", self.storage_pool, substring=True)
```

Comment:
why you need substring=True? You have the full volume name and also the full pool name no?